### DISCUSSION TITLE
How to user Passkeys (Web Auth) in Termux-X11?

### DISCUSSION THREAD

**Discussion by gutierri** [termux/termux-app#4043] [Q&A] [answered]
Is there a possibility for termux X11 to communicate with the device's biometrics and respond to the browser's Passkeys?

**Comment by KebabLord** [comment]
https://wiki.termux.com/wiki/Termux-fingerprint

**Reply by sylirre** [reply]
`termux-fingerprint` is not usable for passkeys via device biometrics. It is a simple fingerprint dialog returning authenticated/non-authenticated response.

**Comment by sylirre** [comment, marked as answer]
@gutierri As for now you can't use on-device passkeys with Termux.

Moreover I'm afraid that this feature will never be implemented because requires seamless integration of X11 app with credential manager API on the Android.

More chances that Yubikey or other USB-based FIDO device support will be added via `termux-usb` or something like.
